Sources/Turf/GeoJSON.swift

```diff
@@ -89,3 +89,36 @@ extension Feature: GeoJSONObjectConvertible {
 extension FeatureCollection: GeoJSONObjectConvertible {
     public var geoJSONObject: GeoJSONObject { return .featureCollection(self) }
 }
+
+/**
+ A GeoJSON object that can contain [foreign members](https://datatracker.ietf.org/doc/html/rfc7946#section-6.1) in arbitrary keys.
+ */
+public protocol ForeignMemberContainer {
+    /// [Foreign members](https://datatracker.ietf.org/doc/html/rfc7946#section-6.1) to round-trip to JSON.
+    var foreignMembers: JSONObject { get set }
+}
+
+extension ForeignMemberContainer {
+    /**
+     Decodes any foreign members using the given decoder.
+     */
+    mutating func decodeForeignMembers<WellKnownCodingKeys>(notKeyedBy _: WellKnownCodingKeys.Type, with decoder: Decoder) throws where WellKnownCodingKeys: CodingKey {
+        let foreignMemberContainer = try decoder.container(keyedBy: AnyCodingKey.self)
+        for key in foreignMemberContainer.foreignKeys(excludingKeysIn: WellKnownCodingKeys.self) {
```

Comment:
This is the only place where `foreignKeys(excludingKeysIn:)` is used. Do we need this as an extension?

Reply:
I inlined the method.